### DISCUSSION TITLE
Update directly in AMP

### DISCUSSION THREAD

**Discussion by L4ma** [MitchTalmadge/AMP-dockerized#176] [Q&A]
Hello everyone,
For some time now, I've been seeing that AMP has an update that I can start directly via the web interface. Is this also possible in the Docker version or do I have to wait until a new Docker image is available? 
![Bildschirmfoto 2023-11-08 um 11 36 45](https://github.com/MitchTalmadge/AMP-dockerized/assets/26965333/7576546c-8a59-4557-a1c7-dc376f2e01f0)

**Comment by MitchTalmadge** [comment]
In our latest update to be released very soon, you can update from the web interface. In addition the container will automatically update on startup by default in the new version. If you would like to try it, the image is `mitchtalmadge/amp-dockerized:staging-debian-12`. 

:)
